### PR TITLE
Update UG with NRIC

### PR DESCRIPTION
**Pull request by linqing42**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -182,10 +183,10 @@ If your changes to the data file makes its format invalid, Police Address Book w
 
 Action | Format, Examples
 --------|------------------
-**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS r/REMARK d/DATE i/NRIC [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 r/shop theft d/12-20-2012 i/S12345678A t/friend t/colleague`
+**Add** | `add n/NAME i/IC p/PHONE_NUMBER e/EMAIL a/ADDRESS r/REMARK d/DATE i/NRIC [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 r/shop theft d/12-20-2012 i/S12345678A t/friend t/colleague`
```
Comment (Impala36) [inline docs/UserGuide.md:186]: Perhaps the second NRIC field should be removed?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -182,10 +183,10 @@ If your changes to the data file makes its format invalid, Police Address Book w
...
 **Clear** | `clear`
 **Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [r/REMARK] [d/DATE] [i/NRIC] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
+**Edit** | `edit INDEX [n/NAME] [i/IC] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [r/REMARK] [d/DATE] [i/NRIC] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
```
Comment (Impala36) [inline docs/UserGuide.md:189]: Same as the comment above.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -182,10 +183,10 @@ If your changes to the data file makes its format invalid, Police Address Book w
...
 **Find** | `find KEYWORD [n/MORE_KEYWORDS] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [r/REMARK] [d/DATE] [i/NRIC] [t/TAG]…`<br> e.g., `find n/James`
```
Comment (Impala36) [inline docs/UserGuide.md:190]: Line 190: Perhaps the NRIC field could also be shifted to after Name?

Comment (linqing42) [reply docs/UserGuide.md:190]: okay

File: docs/UserGuide.md (hunk 2 above)
Comment (linqing42) [reply docs/UserGuide.md:189]: noted

**Review by Impala36 — CHANGES_REQUESTED** [review]
I would also like to suggest to standardize the NRIC for all references, instead of having some as IC.

**Pull request closed without merging**